### ISSUE TITLE
GraphQL not working with block editor field

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#22391]
**Describe the bug**
GraphQL query that include a block editor field is throwing an error and not returning the content of the block editor field.

**To Reproduce**
1. Go to GraphQL playground
2. Run this query
```
query PostBySlug {
  BlogCollection(
    query: "+urlmap:/blog/post/ecotourism-in-costa-rica"
    limit: 1
  ) {
    title
    urlTitle
    blogContent {
      json
    }
  }
}
```
3. See error and the logs

```
13:05:56.648  WARN  execution.SimpleDataFetcherExceptionHandler - Exception while fetching data (/BlogCollection[0]/blogContent) : null
java.lang.NullPointerException: null
	at java.io.StringReader.<init>(StringReader.java:50) ~[?:?]
	at com.dotmarketing.util.json.JSONTokener.<init>(JSONTokener.java:49) ~[dotcms_22.06_999999.jar:?]
	at com.dotmarketing.util.json.JSONObject.<init>(JSONObject.java:302) ~[dotcms_22.06_999999.jar:?]
	at com.dotcms.graphql.datafetcher.StoryBlockFieldDataFetcher.get(StoryBlockFieldDataFetcher.java:19) ~[dotcms_22.06_999999.jar:?]
	at com.dotcms.graphql.datafetcher.StoryBlockFieldDataFetcher.get(StoryBlockFieldDataFetcher.java:11) ~[dotcms_22.06_999999.jar:?]
	at graphql.execution.ExecutionStrategy.fetchField(ExecutionStrategy.java:270) ~[graphql-java-13.0.jar:?]
	at graphql.execution.ExecutionStrategy.resolveFieldWithInfo(ExecutionStrategy.java:198) ~[graphql-java-13.0.jar:?]
	at graphql.execution.AsyncExecutionStrategy.execute(AsyncExecutionStrategy.java:74) ~[graphql-java-13.0.jar:?]
	at graphql.execution.ExecutionStrategy.completeValueForObject(ExecutionStrategy.java:654) ~[graphql-java-13.0.jar:?]
	at graphql.execution.ExecutionStrategy.completeValue(ExecutionStrategy.java:438) ~[graphql-java-13.0.jar:?]
	at graphql.execution.ExecutionStrategy.completeValueForList(ExecutionStrategy.java:533) ~[graphql-java-13.0.jar:?]
	at graphql.execution.ExecutionStrategy.completeValueForList(ExecutionStrategy.java:485) ~[graphql-java-13.0.jar:?]
	at graphql.execution.ExecutionStrategy.completeValue(ExecutionStrategy.java:423) ~[graphql-java-13.0.jar:?]
	at graphql.execution.ExecutionStrategy.completeField(ExecutionStrategy.java:388) ~[graphql-java-13.0.jar:?]
	at graphql.execution.ExecutionStrategy.lambda$resolveFieldWithInfo$0(ExecutionStrategy.java:200) ~[graphql-java-13.0.jar:?]
	at java.util.concurrent.CompletableFuture.uniApplyNow(CompletableFuture.java:680) ~[?:?]
	at java.util.concurrent.CompletableFuture.uniApplyStage(CompletableFuture.java:658) ~[?:?]
```

**Expected behavior**
It should work with any block editor field

**Comment by fmontes** [comment]
Fixed but also find a new issue: https://github.com/dotCMS/core/issues/22420

**Comment by bryanboza** [comment]
Fixed, tested on release-22.06 // Docker // FF
